vendor/teleopti/calendar-data-updater.directive.js

```diff
@@ -0,0 +1,159 @@
+(function() {
```

Comment:
Auto-syncing styleguide with our WFM is actually a double-edged sword. For example, everytime we tries to build Ccc.web, we need to download the latest min.css from github. In Shenzhen, the connection to github is not always fast, and we have to manually disable this grunt task from time to time. So if the styleguide directives are exported through http downloading during building time, I would prefer to have a copy in our WFM project to maintain separately.

Currently Rob has put his directive in WFM/js/global. The styleguide would more be like a manual page for using that directive. I think we could follow this way of managing custom directives.